staging/src/k8s.io/kubectl/pkg/cmd/set/set_env.go

```diff
@@ -207,6 +203,14 @@ func contains(key string, keyList []string) bool {
 	return false
 }
 
+func (o *EnvOptions) keyToEnvName(key string) string {
+	envName := strings.ToUpper(validEnvNameRegexp.ReplaceAllString(key, "_"))
+	if envName != key {
+		fmt.Fprintf(o.ErrOut, "warning: key %s transferred to %s\n", key, envName)
```

Comment:
Do these end up rendered similarly to the way https://kubernetes.io/blog/2020/09/03/warnings/ explains? (colored yellow, and with an initial capital W)?

Reply:
No, it's just normal font color, without colored yellow and an Initial capital W.

The warning style exists in several places. If you search with `fmt.Fprintf(o.ErrOut, "warning: xxxx`, you can find them.

Do we need coordinate them to the way as you mentioned? If so, I am glad to issue another PR to fix them all

Reply:
> [cleanup]

Sounds good to me. How about first filing a kubectl issue to track the work? See https://github.com/kubernetes/kubectl/issues/new/choose

Reply:
Sure, track issue first :-)

Reply:
FYI - https://github.com/kubernetes/kubectl/issues/1177